### DISCUSSION TITLE
How to make personal account visible on the `sponsors/accounts` page

### DISCUSSION THREAD

**Discussion by wu-sheng** [community/community#19884] [Other Feature Feedback, Questions, & Ideas]
I disabled `sponsor` through `Disable GitHub Sponsors account`, https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/disabling-your-github-sponsors-account

How could my personal account be available on my `GitHub Sponsors eligible accounts`? It seems missing.

**Comment by MathiasReker** [comment]
This is not possible

**Reply by wu-sheng** [reply]
Is there a way to enable it? Or GitHub doesn't provide this?
